### PR TITLE
Introduce Command Line Flags

### PR DESCRIPTION
**Pull request by sputnik27**
This commit introduces a way to connect and log-on via command line
flags.

### CODE REVIEW

**Comment by sputnik27** [comment]
I am not sure if I want to move the client secret and the password to a environment variable instead. This is more secure because one can not see it in the list of processes.

**Comment by sputnik27** [comment]
Does anyone of @osiam/owners have a few minutes to review this?

File: gradle.properties [hunk 1]
```diff
@@ -1,2 +1,3 @@
 osiamConnectorVersion=1.7-SNAPSHOT
 jSimpleShellVersion=4.0
+jCommanderVersion=1.48
```
Comment (wallner) [inline gradle.properties]: please add a newline

**Comment by wallner** [comment]
I like the work. I would suggest that we do add a line describing the license and a helpful help text to be displayed when the application is called with the `--help` flag

**Comment by sputnik27** [comment]
> license and a helpful help text

Good idea. I updated my commit accordingly.

**Comment by wallner** [comment]
THank you.

**Pull request merged**